cmake/flash/CMakeLists.txt

```diff
@@ -167,7 +167,7 @@ if(DEFINED ENV{WEST_DIR} AND NOT WEST_DIR)
 endif(DEFINED ENV{WEST_DIR} AND NOT WEST_DIR)
 
 if(WEST_DIR)
-  set(WEST "PYTHONPATH=${WEST_DIR}/src" "${PYTHON_EXECUTABLE};${WEST_DIR}/src/west/main.py;--zephyr-base=$ENV{ZEPHYR_BASE} ")
+  set(WEST "PYTHONPATH=${WEST_DIR}/src" "${PYTHON_EXECUTABLE};${WEST_DIR}/src/west/app/main.py;--zephyr-base=$ENV{ZEPHYR_BASE} ")
```

Comment:
Does this change require us to bump the MIN_WEST_VERSION in host-tools.cmake ?

Reply:
Good catch, yes.

Reply:
A quick glance seem to indicate that versions >= 0.7.0 has this structure. Can the MIN_WEST_VERSION be bumped to 0.7.0?

Reply:
I think 0.7.2 has enough bugfixes compared to 0.7.0 that it's worth switching to it as a minimum if we are going to drop support for 0.6.x.